app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
Not all 'unpublished' items items are 'withdrawn' so some will no longer be 'visible' having been redirected. I'm not sure how you're consuming this and whether that is an issue but thought it would be worth mentioning on account of the comment.